studio-docs/source/explorer/additional-features.mdx

```diff
@@ -74,6 +66,48 @@ You can pause introspection polling at any time. To do so:
 * **In Sandbox**, open the Explorer Settings tab on the left and edit your connection settings. In the dialog, disable **Auto Update**.
 * **In dev graphs**, open the **DEV GRAPH** menu at the top and disable **Auto Update**.
 
+## Federation
+
+### Query plans for federated graphs
+
+If you're working with a federated graph in Studio, the Explorer dynamically calculates example query plans for your operations in the right-hand panel (click **Response** and select **Query Plan Preview** from the dropdown):
+
+<img class="screenshot" src="../img/explorer-query-plan.jpg" alt="Query plan in the Explorer" width="300" />
+
+> If you don't see the **Query Plan Preview** option, make sure you're using the full version of the Apollo Explorer, _not_ Apollo Sandbox. Also make sure you're using the Explorer with a federated graph.
+
+As you edit your operation, the Explorer recalculates the example query plan accordingly.
+
+> **⚠️ The Explorer might not always calculate the exact same query plan as your gateway!** The Explorer's example query plan helps you reason about your operations, but it is _not_ intended as a source of truth.
```

Comment:
Potentially worth specifying that this is because we might be running a different version of the query planner? Don't want to imply that planning is non deterministic

Reply:
Good call! I'll clarify.